source/glossary.rst

```diff
@@ -218,7 +218,10 @@ Glossary
       The Frontier pre-release, which launched on May 9th 2015. It was meant for developers to help test the limits of the Ethereum blockchain.
 
    morden
-      Morden is the first Ethereum alternative testnet. It is expected to continue throughout the Frontier and Homestead era.
+      Morden was the first Ethereum alternative testnet. 
+
+   ropsten
+      Ropsten is a testnet forked from Morden in November 2016 due to increasing mining difficulty, and a client consensus issue.
```

Comment:
Was Ropsten a fork, or a freshly started chain?

Reply:
Good point, I got mixed up since the consensus issue was caused by a fork.

Reply:
Many things happened around that time.